test/src/TRestWimpSensitivity.cxx

```diff
@@ -30,5 +30,8 @@ TEST(TRestWimpSensitivity, FromRml) {
 
     WS.PrintMetadata();
 
-    EXPECT_TRUE(abs(WS.GetSensitivity(1) - 2.9732007e-39) < 1E-45);
+    const double val = WS.GetSensitivity(1);
+    const double expected = 1.12828e-39;
+    cout << "Value " << val << " Expected " << expected << endl;
+    EXPECT_TRUE(abs(val - expected) < 1E-45);
```

Comment:
Its nice to see people are writing tests!

@AlvaroEzq test frameworks usually have dedicated ways to assert equality of built-in types (see suggestion below). If you need additional tolerance you can also use [this one](http://google.github.io/googletest/reference/assertions.html#EXPECT_NEAR).

It is also not necessary to printout the values to test, if the test fails it will print the mismatching values in a clear way.

```suggestion
    EXPECT_DOUBLE_EQ(val, 1.12828e-39);
```

You can find the relevant documentation at http://google.github.io/googletest/reference/assertions.html.